frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/StrategyExecution.tsx

```diff
@@ -243,11 +243,9 @@ export const StrategyExecution: VFC<IStrategyExecutionProps> = ({
     }
 
     const listItems = [
-        Boolean(uiConfig.flags.SE) &&
```

Comment:
Nit: we can remove https://github.com/Unleash/unleash/pull/4690/files#diff-15837dd9de747f632e48d5fb033bab4942c4ebb565c91a8828b50da4bb8ba630R47 and probably the import of useUiConfig as well